source/tests/system/nirfmxinstr_driver_api_tests.cpp

```diff
@@ -125,13 +130,26 @@ TEST_F(NiRFmxInstrDriverApiTests, GetNIRFSASession_SelfTest_Succeeds)
 TEST_F(NiRFmxInstrDriverApiTests, InitializeFromNIRFSA_SelfCalibrate_Succeeds)
 {
   auto rfsa_stub = create_stub<nirfsa_grpc::NiRFSA>();
-  auto init_rfsa_response = nirfsa_client::init_with_options(rfsa_stub, "Sim", false, false, "Simulate=1, DriverSetup=Model:5663E");
+  auto init_rfsa_response = init_rfsa(rfsa_stub, "Sim");
   ni::tests::system::EXPECT_SUCCESS(init_rfsa_response);
   auto init_response = client::initialize_from_nirfsa_session(stub(), init_rfsa_response.vi());
   auto session = init_response.instrument();
   EXPECT_SUCCESS(session, init_response);
 
-  client::self_calibrate(stub(), session, "", 0);
+  EXPECT_SUCCESS(session, client::self_calibrate(stub(), session, "", 0));
+}
+
+TEST_F(NiRFmxInstrDriverApiTests, InitializeFromNIRFSAArray_SelfCalibrate_Succeeds)
+{
+  auto rfsa_stub = create_stub<nirfsa_grpc::NiRFSA>();
+  auto init_rfsa_response = init_rfsa(rfsa_stub, "Sim");
+  auto init_rfsa_response_2 = init_rfsa(rfsa_stub, "Sim2");
+  ni::tests::system::EXPECT_SUCCESS(init_rfsa_response);
```

Comment:
Minor: Should we go ahead and add a matching `EXPECT_SUCCESS` for the second init rsponse?